src/renderer/components/QueryResultNav/QueryResultNav.css

```diff
@@ -55,7 +58,6 @@
   border: 1px solid #ccc;
   background-color: #fff;
   border-radius: 4px;
-  z-index: 1;
```

Comment:
When using `position: sticky`, z-index of child node does not work fine.
https://stackoverflow.com/a/53107499/6945346